### DISCUSSION TITLE
Buzzing sound from HDMI speakers when pressing any icon button in music assistant when run from sidebar

### DISCUSSION THREAD

**Discussion by harryfine** [music-assistant/support#3355] [Q&A]
Call me crazy, I didn't believe it myself.

I love and rely on MA.  Newest version of server and integration, 2.4.0b8 using native HA integration on HAOS, newest 2024.12.x version of HA.

Was doing some playing around in the speaker settings, grouping, ungrouping, testing really because a new version pushed today.  I was hearing a quiet hiss sound.  My SONOS speakers were playing correctly as selected, but this quiet hiss sound at my computer.  Unplugged my Google little speaker at my desk, it's not part of MA but I thought I would try.  No different.  I listened more closely.  The hiss was coming from my HDMI speakers on my Dell monitor, my default output on my Linux Mint system.

So started to play around:

1. No music playing --> Start MA from the sidebar
2. Go into speakers by selecting the little speaker icon on the main screen
3. Hiss starts
4. Went into speaker select --> selected a different one
5. Back to main screen.  Music plays correctly.  Hiss still going.
6. Exit MA back to main dashboard --> Hiss stops, all good

So tried a different tactic:

1. No music playing --> Start MA from my default dashboard which has buttons including one for MA as a web card
2. Go into speakers by selecting the little speaker icon on the main screen
3. No Hiss

Restarted computer.  Restarted HA.  Tried above.  Same result. Then by chance I clicked another button.  Settings.  Buzz started.  Exited to main dashboard it stopped.  Started it again from sidebar.  Pressed play or browse or whatever, same thing.

What did starting from the sidebar create this audio condition?

**Comment by OzGav** [comment]
Did this start with the latest beta or you just noticed it now.

**Comment by harryfine** [comment]
Just noticed it, and the buzz is so soft it is possible it was there
before.  

-----Original Message-----
From: OzGav ***@***.***>
Reply-To: music-assistant/hass-music-assistant
***@***.***>
To: music-assistant/hass-music-assistant
***@***.***>
Cc: Harry Fine ***@***.***>, Author ***@***.***>
Subject: Re: [music-assistant/hass-music-assistant] Buzzing sound from
HDMI speakers when pressing any icon button in music assistant when run
from sidebar (Discussion #3355)
Date: 2024-12-23 10:25:54 AM

Did this start with the latest beta or you just noticed it now.
—
Reply to this email directly, view it on GitHub [1], or unsubscribe
[2].
You are receiving this because you authored the thread.Message ID:
<music-assistant/hass-music-assistant/repo-
***@***.***>


[1] view it on GitHub
    https://github.com/orgs/music-assistant/discussions/3355#discussioncomment-11651164
[2] unsubscribe
